### ISSUE TITLE
if a user enters the same variable in the debug console and the value hasn't changed, the value isn't announced

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#50078]
1. start debugging and enable a screen reader
2. on a breakpoint, enter a var name in the debug console
3. you'll hear it's value ✔ 
4. without continuing the debug session, enter it again
5. 🐛 screen reader doesn't announce the value
